homeassistant/components/ecobee/sensor.py

```diff
@@ -112,7 +112,7 @@ def device_class(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._state in [ECOBEE_STATE_CALIBRATING, ECOBEE_STATE_UNKNOWN]:
+        if self._state in [ECOBEE_STATE_CALIBRATING, ECOBEE_STATE_UNKNOWN, "unknown"]:
```

Comment:
Should this become part of PYECOBEE ?

Reply:
For sure it should. There are a bunch of additions to pyecobee I want to make when the dust has settled on the changes in HA.